docs/source/tutorials/qubo_problem.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
<p>indicate what the legend (1, 0.25, 0.1) mean in both graphs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>